package.json

```diff
@@ -13,7 +13,7 @@
   "typings": "lib/analyzer.d.ts",
   "scripts": {
     "clean": "mkdir -p lib/ && rm -rf lib/ && mkdir -p lib/test && cp -r src/test/static lib/test/",
-    "build": "tsc",
+    "build": "tsc || true",
```

Comment:
This was required to not crash `npm test` because `tsc` does not like `npm link` as you mentioned on Slack. If you remove this line, it will once again break the run. Should I add it somewhere else?